Blocks/HTTP/impl/posix_server.h

```diff
@@ -258,6 +303,29 @@ class HTTPServerPOSIX final {
     }
   }
 
+  HTTPRoutesScopeEntry DoRegisterHandler(const std::string& path,
+                                         std::function<void(Request)> handler,
+                                         const URLPathParams::Count path_param_count,
+                                         const ReRegisterRoute policy) {
+    if (path.empty() || path[0] != '/' || (path.length() > 1 && path[path.length() - 1] == '/') ||
+        !URL::IsValidPath(path)) {
+      CURRENT_THROW(InvalidHandlerPathException("Invalid handler path '" + path + "'"));
```

Comment:
Done in https://github.com/c5t/Current/commit/91cd01b19758c0a3e67b3f5be4d77caf10213bd6